packages/react-dom/src/__tests__/ReactDOMServerIntegrationModes-test.js

```diff
@@ -39,6 +39,53 @@ describe('ReactDOMServerIntegration', () => {
     resetModules();
   });
 
+  // Test pragmas don't support itRenders abstraction
+  if (require('shared/ReactFeatureFlags').enableDebugTracing) {
```

Comment:
Can we align feature flag closer to component name? One way or another. I keep mistyping either of them because "trace" vs "tracing". :D

Reply:
Haha, yeah sure thing